src/Common/Select.php

```diff
@@ -612,12 +612,14 @@ public function groupBy(array $spec)
      *
      * @param string $cond The HAVING condition.
      *
+     * @param array ...$bind arguments to bind to placeholders
```

Comment:
it can't be `string` because it may not always be a string. https://github.com/auraphp/Aura.SqlQuery/blob/3.x/src/AbstractQuery.php#L403. Is `mixed` better than a union type eg `string|SubselectInterface`?